src/index.html

```diff
@@ -1,12 +1,414 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+      content="width=device-width,
+      initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="page__header header">
+      <a href="#home" class="header__logo">
+        <img
+          src="/images/logo.png"
+          alt="Main logo"
```

Comment:
```suggestion
          alt="main logo"
```
не супер кримінал, але пишеться з маленької)